dev/fattest.simplicity/src/componenttest/topology/utils/MvnUtils.java

```diff
@@ -979,12 +981,64 @@ public static int runCmd(String[] cmd, File workingDirectory, File outputFile) t
         pb.redirectErrorStream(true);
 
         Log.info(c, "runCmd", "Running command " + Arrays.asList(cmd));
-        Process p = pb.start();
-        int exitCode = p.waitFor();
-        return exitCode;
+        
+        int hardTimeout;
+        
+        try {
+            hardTimeout = Integer.parseInt(System.getProperty("fat.timeout"));
```

Comment:
instead of catching an exception here you can use `System.getProperty("fat.timeout", "10800000")` where the 2nd param is the default value if no sys prop is found